speed-dreams/source-2.2.3/src/libs/tgfclient/guiscrollist.cpp

```diff
@@ -651,7 +651,7 @@ gfuiDrawScrollist(tGfuiObject *obj)
 				break;
 			}
 			y -= h;
-			gfuiDrawString(x+5, y, scrollist->font, elt->label);
+            gfuiDrawString(x+5, y, scrollist->font, elt->label);
```

Comment:
Dont push a changed SD file with only some spaces as difference

Reply:
Oh Visual Studio automatically did this when I renamed this variable at first (will fix it now)